http/httpclient/src/main/java/com/okta/commons/http/httpclient/HttpClientRequestExecutor.java

```diff
@@ -70,36 +72,29 @@
 
     private static final Logger log = LoggerFactory.getLogger(HttpClientRequestExecutor.class);
 
-    private static final int DEFAULT_MAX_CONNECTIONS_PER_ROUTE = Integer.MAX_VALUE/2;
     private static final String MAX_CONNECTIONS_PER_ROUTE_PROPERTY_KEY = "com.okta.sdk.impl.http.httpclient.HttpClientRequestExecutor.connPoolControl.maxPerRoute";
-    private static final int MAX_CONNECTIONS_PER_ROUTE = parseConfigValue(MAX_CONNECTIONS_PER_ROUTE_PROPERTY_KEY,
-                                                                          DEFAULT_MAX_CONNECTIONS_PER_ROUTE,
-                                                                         "Bad max connection per route value");
-
-    private static final int DEFAULT_MAX_CONNECTIONS_TOTAL = Integer.MAX_VALUE;
     private static final String MAX_CONNECTIONS_TOTAL_PROPERTY_KEY = "com.okta.sdk.impl.http.httpclient.HttpClientRequestExecutor.connPoolControl.maxTotal";
-    private static final int MAX_CONNECTIONS_TOTAL = parseConfigValue(MAX_CONNECTIONS_TOTAL_PROPERTY_KEY,
-                                                                      DEFAULT_MAX_CONNECTIONS_TOTAL,
-                                                              "Bad max connection total value");
-
-    private static final int DEFAULT_CONNECTION_VALIDATION_INACTIVITY = 2000; // 2sec
     private static final String CONNECTION_VALIDATION_PROPERTY_KEY = "com.okta.sdk.impl.http.httpclient.HttpClientRequestExecutor.connPoolControl.validateAfterInactivity";
-    private static final int CONNECTION_VALIDATION_INACTIVITY = parseConfigValue(CONNECTION_VALIDATION_PROPERTY_KEY,
-                                                                                 DEFAULT_CONNECTION_VALIDATION_INACTIVITY,
-                                                                                "Invalid max connection inactivity validation value");
+    private static final String CONNECTION_TIME_TO_LIVE_PROPERTY_KEY = "com.okta.sdk.impl.http.httpclient.HttpClientRequestExecutor.connPoolControl.timeToLive";
 
+    private static final String DEFAULT_REQUEST_EXECUTOR_MAX_CONNECTIONS_PER_ROUTE_PROPERTY_NAME = "okta.client.requestExecutor.maxPerRoute";
```

Comment:
This prefix could be anything, the `okta.client.*` is specific to the SDK, but for our AUTHN or JWT libraries, these prefixes would be different.
Those libraries should only pass the properties in (stripping the prefix)

The legacy `com.okta.sdk.impl` system properties have `sdk` in the name for backward compatibility reasons.